torch/csrc/autograd/python_function.cpp

```diff
@@ -1361,6 +1368,7 @@ static struct PyGetSetDef THPFunction_properties[] = {
 // NOLINTNEXTLINE(modernize-avoid-c-arrays,cppcoreguidelines-avoid-c-arrays,cppcoreguidelines-avoid-non-const-global-variables)
 static struct PyMethodDef THPFunction_methods[] = {
     {(char*)"name", THPFunction_name, METH_NOARGS, nullptr},
+    {(char*)"sequence_nr", THPFunction_sequence_nr, METH_NOARGS, nullptr},
```

Comment:
We probably want these to be private API for now, i.e. we should expose it as `_sequence_nr` as what it returns is not super straightforward for users

Reply:
Makes sense - I changed it to _sequence_nr in both python_function.cpp and python_cpp_function.h